app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -44,11 +45,27 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   searchResult: {
+    marginBottom: theme.spacing(3),
+    borderRadius: theme.shape.borderRadius * 2,
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: "0 0 4px rgba(0,0,0,0.25)",
     [theme.breakpoints.down("sm")]: {
-      maxWidth: "80%",
-    },
-    [theme.breakpoints.up("md")]: {
-      margin: theme.spacing(2),
+      padding: 0,
+      overflow: "hidden",
+      flexShrink: 0,
+      width: "85vw",
+      maxWidth: "33rem",
+      margin: theme.spacing(0, 2, 0, 0),
+      scrollSnapAlign: "start",
+      "&:last-child": {
+        marginRight: 0,
+      },
+      "& MuiCardContent-root": {
+        padding: 0,
+        height: `calc(100% - ${theme.spacing(4)})`,
+        overflow: "hidden",
+        margin: theme.spacing(2),
+      },
```

Comment:
Invalid selector, looks fine though so remove this?

Reply:
It was a fix for overlap happening in narrow screens. Should be working with last commit.